src/main/java/org/fundacionjala/sfdc/util/PropertiesSalesForce.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.sfdc.util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.fundacionjala.sfdc.pageobjects.login.SalesForceLogIn;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.pageobjects.login.SalesForceLogIn'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)